roles/common/README.md

```diff
@@ -156,10 +171,42 @@ can be set at the play level.
           name: common
         vars:
           common_project_test_id: "RHOSO-12668"
+          common_service_nspace: openshift-logging
```

Comment:
this need to be with the Verify Service example

Reply:
Done